test/color/setting_test.dart

```diff
@@ -180,6 +180,68 @@ void main() {
 
         await screenMatchesGolden(tester, 'color_setting_nostroke-example-1');
       });
+
+      processingLegacySpecTest('clear() replaces background in setup() and drawn rect', (tester) async {
+        await tester.pumpWidget(
+          Processing(
+            sketch: Sketch.simple(
+              setup: (s) {
+                s.background(color: const Color(0xFF404040));
+              },
+              draw: (s) {
+                s
+                  ..noLoop()
+                  ..fill(color: const Color(0xFF969696))
+                  ..rect(rect: Rect.fromLTWH(30, 20, 55, 55))
+                  ..clear();
+              },
+            ),
+          ),
+        );
+
+        await screenMatchesGolden(tester, 'color_setting_clear-draw-then-clear');
+      });
+
+      processingLegacySpecTest('clear() replace background in setup(), then draw rect on transparent background',
+          (tester) async {
+        await tester.pumpWidget(
+          Processing(
+            sketch: Sketch.simple(
+              setup: (s) {
+                s.background(color: const Color(0xFF404040));
+              },
+              draw: (s) {
+                s
+                  ..noLoop()
+                  ..clear()
+                  ..fill(color: const Color(0xFF969696))
+                  ..rect(rect: Rect.fromLTWH(30, 20, 55, 55));
+              },
+            ),
+          ),
+        );
+
+        await screenMatchesGolden(tester, 'color_setting_clear-clear-then-draw');
```

Comment:
This test name has two "clears": `_clear-clear`. That seems redundant.

Reply:
I meant the first `clear` to be the group name, which is `color_setting_clear`, the second `clear` is for action